public/app/features/alerting/unified/utils/query.ts

```diff
@@ -0,0 +1,64 @@
+import { DataQuery, DataSourceInstanceSettings } from '@grafana/data';
+import { LokiQuery } from 'app/plugins/datasource/loki/types';
+import { PromQuery } from 'app/plugins/datasource/prometheus/types';
+import { CombinedRule } from 'app/types/unified-alerting';
+import { AlertQuery } from 'app/types/unified-alerting-dto';
+import { isCloudRulesSource, isGrafanaRulesSource } from './datasource';
+import { isGrafanaRulerRule } from './rules';
+
+export function alertRuleToQueries(combinedRule: CombinedRule | undefined | null): AlertQuery[] {
+  if (!combinedRule) {
+    return [];
+  }
+
+  const { namespace, rulerRule } = combinedRule;
+  const { rulesSource } = namespace;
+
+  if (isGrafanaRulesSource(rulesSource)) {
+    if (isGrafanaRulerRule(rulerRule)) {
+      return rulerRule.grafana_alert.data;
+    }
+  }
+
+  if (isCloudRulesSource(rulesSource)) {
+    const model = cloudAlertRuleToModel(rulesSource, combinedRule);
+
+    return [
+      {
+        refId: model.refId,
+        datasourceUid: rulesSource.uid,
+        queryType: '',
+        model,
```

Comment:
```suggestion
        model,
        relativeTimeRange: {
          from: 60 * 60,
          to: 0,
        },
```

if we show a time series, we need a time range. Right now explore will show a graph, but this view shows a single datapoint at the last coordinate